src/importexport/musicxml/internal/musicxml/importmxmlpass1.cpp

```diff
@@ -1088,6 +1088,9 @@ Err MusicXMLParserPass1::parse()
 
     if (!found) {
         m_logger->logError(u"this is not a MusicXML score-partwise file, node <score-partwise> not found", &m_e);
+        if (!m_e.errorString().isEmpty()) {
+            m_errors += errorStringWithLocation(m_e.lineNumber(), m_e.columnNumber(), m_e.errorString()) + '\n';
+        }
```

Comment:
ensure at least some useful error info shown if score-partwise can't be detected due to an unrecoverable low-level parsing error